core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -132,6 +133,8 @@ class KafkaApisTest {
       val properties = TestUtils.createBrokerConfig(brokerId, "")
       properties.put(KafkaConfig.NodeIdProp, brokerId.toString)
       properties.put(KafkaConfig.ProcessRolesProp, "broker")
+      val nodeIdMod = (brokerId + 1)
+      properties.setProperty(RaftConfig.QUORUM_VOTERS_CONFIG, s"${nodeIdMod.toString}@localhost:9093")
```

Comment:
String interpolation should call `toString`. This comment applies to a few places.